code/game/objects/effects/glowshroom.dm

```diff
@@ -65,12 +65,12 @@
 	else
 		myseed = new myseed(src)
 	if(spread)
-		myseed.potency -= round(myseed.potency * 0.25) // Reduce potency of the little mushie if it's spreading
+		myseed.adjust_potency(-round(myseed.potency * 0.25)) // Reduce potency of the little mushie if it's spreading
 	if(mutate_stats) //baby mushrooms have different stats :3
 		myseed.adjust_potency(rand(-4,3))
 		myseed.adjust_yield(rand(-3,2))
 		myseed.adjust_production(rand(-3,3))
-		myseed.endurance = clamp(myseed.endurance + rand(-3,2), 0, 100) // adjust_endurance has a min value of 10, need to edit directly
+		myseed.endurance = clamp(myseed.endurance + rand(-3,2), 0, MAX_PLANT_ENDURANCE) // adjust_endurance has a min value of 10, need to edit directly
```

Comment:
you dont have to do this but the fact that adjust_endurance has a min value of 10 is kind of cringe, if you feel like it feel free to make that less crap because everything should just use the wrapper.

Reply:
so every plant uses a minimum endurance of 10 with the only exception of planted structure growshrooms, so I'm just gonna kick the planted glowshrooms to use 10 as well (functionally pretty much the same) so the wrappers can be used but in the future the proc can be edited to accommodate for this kinda stuff